libs/hpc-ui/src/lib/components/form-fields/autocomplete-field.tsx

```diff
@@ -0,0 +1,65 @@
+import { Autocomplete, AutocompleteProps } from '@mui/material';
+import { useField, useFormikContext } from 'formik';
+import tw from 'twin.macro';
+import { StyledTextField } from './text-field';
+import { FormObjectValue } from '@unocha/hpc-data';
+
+const StyledAutocomplete = tw(Autocomplete)`
+      min-w-[10rem]
+      w-full`;
```

Comment:
Would sure look nicer if these are formatted like rest of the code:
```suggestion
const StyledAutocomplete = tw(Autocomplete)`
  min-w-[10rem]
  w-full
`;
```